Intersect.Server.Core/Maps/MapInstance.cs

```diff
@@ -144,11 +145,15 @@ public partial class MapInstance : IDisposable
         private MapActionMessages mActionMessages = new MapActionMessages();
         private MapAnimations mMapAnimations = new MapAnimations();
 
-        public MapInstance(MapController map, Guid mapInstanceId)
+        public MapInstance(MapController map, Guid mapInstanceId, Player creator)
         {
             mMapController = map;
             MapInstanceId = mapInstanceId;
             Id = Guid.NewGuid();
+            if (!InstanceProcessor.CurrentControllers.Contains(MapInstanceId))
```

Comment:
AddInstanceController already makes this check

Reply:
Fixed - changed to `Try` so that the check is more implied by the name